CourseManager/Courses/Course 4 UI and Animations in SwiftUI/Views/Components/WeatherWidget.swift

```diff
@@ -0,0 +1,66 @@
+//
+//  WeatherWidget.swift
+//  CourseManager
+//
+//  Created by Summer Crow on 2022-11-10.
+//
+
+import SwiftUI
+
+struct WeatherWidget: View {
+    var forecast: Forecast
+    
+    var body: some View {
+        ZStack(alignment: .bottom) {
+            // MARK: Trapezoid
+            Trapezoid()
+                .fill(Color.weatherWidgetBackground)
+                .frame(width: 342, height: 174)
```

Comment:
In almost all cases you can go without fixed frames, so try to update the code to note use them. The main reason is that the screen isn't adaptable to other devices screens